src/amo/components/Footer/index.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import LanguagePicker from 'amo/components/LanguagePicker';
+import translate from 'core/i18n/translate';
+import Icon from 'ui/components/Icon';
+
+import './styles.scss';
+
+
+export class FooterBase extends React.Component {
+  static propTypes = {
+    handleViewDesktop: PropTypes.func.isRequired,
+    i18n: PropTypes.object.isRequired,
+    lang: PropTypes.string.isRequired,
+    location: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { handleViewDesktop, i18n, lang, location } = this.props;
+    const homepageText = i18n.gettext("Go to Mozilla's homepage");
+
+    return (
+      <footer className="Footer">
+        <div className="Footer-language-and-links">
+          <LanguagePicker location={location}
+            ref={(ref) => { this.languagePicker = ref; }} />
+          <ul className="Footer-links">
+            <li>
+              <a href={`https://www.mozilla.org/${lang}/privacy/websites/`}
```

Comment:
Yeah it's a shame that sometimes it falls apart. I know MDN in the past suffered from problems if the locale was one it didn't know.